Mcs 2103 device management i18n

dariachen opened the pull request:
Hi @anchi1216 , @appleboy , @evenchange4 and @MTK-mcs 
Please help me to review these translation for device management. Thanks!

dariachen committed:
Translate the document into zh-TW

dariachen committed:
1. Typo in beta-release.md
2. Translate getting start guide
1. Typo in beta-release.md
2. Translate getting start guide
1. Typo in beta-release.md
2. Translate getting start guide
1. Typo in beta-release.md
2. Translate getting start guide
1. Typo in beta-release.md
2. Translate getting start guide
1. Typo in beta-release.md
2. Translate getting start guide
1. Typo in beta-release.md
2. Translate getting start guide
1. Typo in beta-release.md
2. Translate getting start guide
1. Typo in beta-release.md
2. Translate getting start guide
1. Typo in beta-release.md
2. Translate getting start guide
1. Typo in beta-release.md
2. Translate getting start guide

dariachen committed:
1. Modify wording "Beta裝置" to "裝置"
2. Modify some typo

dariachen committed:
Translate the following 3 files into zh-CN.

appleboy commented:
@dariachen Please follow Chinese Copywriting Guidelines.

https://github.com/sparanoid/chinese-copywriting-guidelines

appleboy commented on content/zh-CN/api_references/device_activation.md:
```diff
@@ -1,109 +1,109 @@
+        }
+    ]
+    “的StatusCode”：400，
+    “选择”：{}
```
Don't translate source code.

anchi1216 commented on content/zh-CN/api_references/device_activation.md:
```diff
@@ -1,109 +1,109 @@
 | --- | --- | --- |
-| code | Integer | Error Code |
-| message | String | Error Description |
+|代码|整数|错误代码|
```
code, message 等 API 回來的東西和 Type, 不需要翻譯喔!

anchi1216 commented on content/zh-CN/api_references/device_activation.md:
```diff
@@ -1,109 +1,109 @@
+    “DEVICEID”：“JU5z4yVF”
+    “deviceKey”：“yQGymH0IRWFSVC37”，
+    “chipName”：“装置物联网雏形”
 }
```
API response 不需要翻譯, DEVICEID 為 deviceId, 另外 deviceKey 最好可以更換為假的喔~

anchi1216 commented on content/zh-CN/api_references/device_activation.md:
````diff
@@ -1,109 +1,109 @@
 ```
-Content-Type:`application/json`
+内容类型：`应用/ json`
 ```
````
Content-Type: application/json 此類型不用翻

anchi1216 commented on content/zh-CN/api_references/device_activation.md:
```diff
@@ -1,109 +1,109 @@
-### Response Header
-For JSON response:
+###回覆头
+对于JSON响应：
```
這邊建議可以參考一下, 其他文件的翻譯喔, 盡量一致

anchi1216 commented on content/zh-TW/tutorial/beta-release.md:
```diff
@@ -1,66 +1,62 @@
 
-## How to create device
+在 **管理** 頁面中的裝置列表是按照產品原型分類，開發者可以在此管理所創建的裝置。此外，每個裝置皆有配發一個獨有的啟用碼，在開始使用此裝置之前使用者必須先完成裝置的啟用程序。
 
```
小小建議, api reference 是翻譯成 激活, 這邊最好也對應把"啟用" 改成 "激活" 會比較一致喔~

dariachen committed:
1. don't translate the sample code into Chinese.
2. follow the guideline to have proper space between English and Chinese characters.

appleboy commented on content/zh-CN/api_references/device_activation.md:
```diff
@@ -72,24 +72,24 @@ Response Body
+當有錯誤發生時，回傳非 200 之其他回覆代碼。回覆內容為 JSON 格式並包含以下資訊：
 
-| Field Name | Type |Description |
+|栏位名称|资料型态|描述|
```
這邊統一格式 | xxxx |

`|` 左右邊各留一個空白

appleboy commented on content/zh-CN/api_references/device_activation.md:
```diff
@@ -72,24 +72,24 @@ Response Body
-| code | Integer | Error Code |
-| message | String | Error Description |
+| code | Integer |错误代码|
+| message | String|错误说明|
```
這邊統一格式 | xxxx |

`|` 左右邊各留一個空白

dariachen committed:
Leave space between character and |.

anchi1216 commented:
look good!

appleboy commented:
LGTM.
